src/components/molecules/native-select/index.tsx

```diff
@@ -0,0 +1,63 @@
+import * as RadixSelect from "@radix-ui/react-select"
+import clsx from "clsx"
+import React from "react"
+import CheckIcon from "../../fundamentals/icons/check-icon"
+import ChevronDownIcon from "../../fundamentals/icons/chevron-down"
+import ChevronUpIcon from "../../fundamentals/icons/chevron-up"
+
+type NativeSelectType = React.FC<NativeSelectProps> & {
+  Item: React.FC<ItemProps>
+}
+
+type NativeSelectProps = {
+  triggerProps?: RadixSelect.SelectTriggerProps
+} & RadixSelect.SelectProps
+
+const NativeSelect: NativeSelectType = ({
+  children,
+  triggerProps,
+  ...props
+}) => {
+  return (
+    <RadixSelect.Root {...props}>
+      <RadixSelect.SelectTrigger
+        className="flex items-center inter-base-semibold gap-3 px-2"
+        {...triggerProps}
+      >
+        <RadixSelect.Value />
+        <RadixSelect.Icon>
+          <ChevronDownIcon size={16} />
+        </RadixSelect.Icon>
+      </RadixSelect.SelectTrigger>
+      <RadixSelect.Content className="rounded-rounded scrollbar-hide border px-2 py-2 border-grey-20 bg-grey-0 w-full flex shadow-dropdown">
```

Comment:
can we add a height property to the Native select component(the time picker has an existing height of 305px)? 

Otherwise the select can grow to the height of the entire screen:
![image](https://user-images.githubusercontent.com/88927411/161231015-b1c0c6ec-68be-4a02-81d7-96b0aa62e3c7.png)

Reply:
Also should we consider removing the `RadixSelect.ScrollDownButton` and `ScrollUpButton`. They dont seem to be in the design but I don't know if you've discussed this with Ludvig.

Reply:
> can we add a height property to the Native select component(the time picker has an existing height of 305px)?

It's actually a bit tricky unfortunately. I think that the select component is supposed to behave that way. It mimics the behavior of native selects in macOS. Unfortunately, giving a max height to the select causes some positioning issues. There is an interesting discussion about this [here](https://github.com/radix-ui/primitives/issues/1193) where the radix team share their thoughts around this. Also, there might be support for setting a max height somewhere down the line (see this [answer](https://github.com/radix-ui/primitives/issues/1193#issuecomment-1050796688)). 

Personally, I think the behavior is not too bad and very acceptable imo, but I agree that being able to set a max-height would be ideal


> Also should we consider removing the RadixSelect.ScrollDownButton and ScrollUpButton. They dont seem to be in the design but I don't know if you've discussed this with Ludvig.

Totally! I thought it was pretty neat from a UX perspective, but will definitely run it by Ludvig and see what he thinks.

Reply:
In that case we should perhaps to something like +- 20 years 😄